extensions/typescript-language-features/package.json

```diff
@@ -1110,6 +1110,12 @@
           "default": "allOpenProjects",
           "markdownDescription": "%typescript.workspaceSymbols.scope%",
           "scope": "window"
+        },
+        "typescript.suggest.includeCompletionsWithClassMemberSnippets": {
```

Comment:
> I didn't add one for JavaScript because we are not supporting this completion feature in JS yet
But it'll come to JS sometime in the future, right?

Reply:
I think supporting JS would be nice. I created an issue to track that, feel free to upvote it: https://github.com/microsoft/TypeScript/issues/46738